solutions/chapter8/services/theme.ts

```diff
@@ -0,0 +1,196 @@
+import {
+  DarkTheme,
+  DefaultTheme,
+  useTheme as useNavTheme,
+} from '@react-navigation/native'
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react'
+import type { StyleProp } from 'react-native'
+import { Appearance, useColorScheme } from 'react-native'
+
+import { changeHexAlpha, colors as lightColors } from '../../../shared/theme'
+
+const darkColors = {
+  background: {
+    primary: '#000000',
+    secondary: '#1a1a1a',
+    accent: '#099faa',
+    accentMuted: '#f8baf7',
+    brand: '#d4f20d',
```

Comment:
I think we should choose an alternate "brand" color that plays more nicely with the light text. 

Like #586600
![CleanShot 2025-01-02 at 13 01 13@2x](https://github.com/user-attachments/assets/6dd488a8-363e-4d79-aab5-59c3a130b79b)

Reply:
great point, adjusting now